engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -210,7 +227,18 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
   onTransition {
     case _ -> toState if toState.terminal =>
```

Comment:
@geoffjentry what I was trying to say what if above case can be broken in onTransition  { 
case _ -> toState if toState == WorkflowExecutionSuccessfulState => ....
case _  -> toState if toState == WorkflowExecutionAbortedState => .... 
case fromState -> toState  => ....
}